source/message/OP_MSG.rst

```diff
@@ -170,20 +172,20 @@ and MUST ignore all undefined optional bits.
 
 The currently defined flags are:
 
-===== ==================== ========= ========== =========================== 
+===== ==================== ========= ========== ===========================
```

Comment:
Please exclude formatting changes.  If you still want to do them, make them a separate commit.

Reply:
Apologies, this came in due to my IDE truncating trailing spaces. I've extracted the changes to a separate commit.

Reply:
Sorry, still seeing the whitespace.